spec/unit/mutant/env/boostrap_spec.rb

```diff
@@ -75,6 +76,11 @@
     context 'when Module#name calls result in exceptions' do
       let(:object_space_modules) { [invalid_class] }
 
+      let(:expected_warning) do
+        "Class#name from: #{invalid_class} raised an error: " \
```

Comment:
Below you use `invalid_class.inspect` but not here. Any reason for the difference?